core/container/src/test/java/org/wildfly/swarm/container/runtime/usage/UsageCreatorTest.java

```diff
@@ -0,0 +1,48 @@
+package org.wildfly.swarm.container.runtime.usage;
```

Comment:
This is missing a copyright too. Not sure how strict you all are about that though.

Reply:
All the new classes are missing a copyright tbh ;)